includes/API.php

```diff
@@ -55,11 +55,15 @@ public function __construct( $access_token ) {
 	 * @since 2.0.0-dev.1
 	 *
 	 * @param $page_id page ID
-	 * @return
+	 * @return API\Pages\Read\Response
```

Comment:
```suggestion
	 * @return API\Pages\Read\Response
	 * @throws Framework\SV_WC_API_Exception
```

seems `perform_request()` is intended to throw an exception in framework - we might not implement throws, but if the response is a WP_Error it still might?   just double checking

Reply:
Good catch @unfulvio! You're right, the framework throws an exception if we get `WP_Error` and we know throw exceptions too, so your suggestion is spot on. Fixed in 228a592.